azure-functions/create-function-app-connect-to-storage/create-function-app-connect-to-storage-account.sh

```diff
@@ -0,0 +1,11 @@
+# login to azure 
+az login
```

Comment:
`az login` is done as a step in the wrapper and doesn't need to be captured here.